crates/transaction-emitter-lib/src/emitter/submission_worker.rs

```diff
@@ -225,9 +227,13 @@ impl SubmissionWorker {
             sample!(
                 SampleRate::Duration(Duration::from_secs(120)),
                 warn!(
-                    "[{:?}] Transactions were not committed before expiration: {:?}",
+                    "[{:?}] Transactions were not committed before expiration: {:?}, for {:?}",
                     self.client.path_prefix_string(),
-                    num_expired
+                    num_expired,
+                    self.accounts
```

Comment:
This is going to spam the logs with many accounts, is this really necessary? If it is, let's sample this?